examples/podman_local/client/feature_repo/test.py

```diff
@@ -0,0 +1,123 @@
+import subprocess
+from datetime import datetime
+import pandas as pd
+from feast import FeatureStore
+from feast.data_source import PushMode
+
+def run_demo():
+    try:
+        store = FeatureStore(repo_path=".")
+
+        print("\n--- Historical features for training ---")
+        fetch_historical_features_entity_df(store, for_batch_scoring=False)
+
+        print("\n--- Historical features for batch scoring ---")
+        fetch_historical_features_entity_df(store, for_batch_scoring=True)
+
+        print("\n--- Load features into online store ---")
+        store.materialize_incremental(end_date=datetime.now())
```

Comment:
I'm tryng the same with a K8s environment and I believe it can't work, as the online server does not implement the remote method `online_write_batch`.
Have you maybe executed the materialization from within a Podman container?

Reply:
This is output I am receiving while running Materializing```--- Load features into online store ---
Materializing 2 feature views to 2024-09-16 12:16:50-04:00 into the remote online store.

driver_hourly_stats from 2024-09-15 12:16:50-04:00 to 2024-09-16 12:16:50-04:00:
0it [00:00, ?it/s]
driver_hourly_stats_fresh from 2024-09-15 12:16:50-04:00 to 2024-09-16 08:16:50-04:00:
0it [00:00, ?it/s]
``` it show 0it it's not working as podman too

Reply:
@tokoko any thought some of the reason materialization is not successful

Reply:
Yes, by running client.

Reply:
@tokoko I think that also remote apply does not work. 
Some APIs are performed on the client side and cannot work. E.g. when using a postgres template I found that the client was running the inference functions from the DB and complained that `RemoteOfflineConfig` missed the `user` property requested to create the DB connection.
Is there another issue to track this request?

Reply:
https://github.com/feast-dev/feast/issues/4529